pyproject.toml

```diff
@@ -25,32 +26,33 @@ classifiers = [
     "Operating System :: Unix",
     "Operating System :: MacOS",
 ]
-requires-python = ">=3.9"
-dependencies = [
-    "aws-cdk-lib==2.48.0",
-    "aws-cdk.aws-lambda-python-alpha",
-    "constructs>=10.0.0,<11.0.0",
-    "boto3",
-    "opensearch-py",
-    "requests",
-    "python-jose"
-]
 
-[project.optional-dependencies]
-dev = [
-    "pytest==6.2.5",
-    "moto"
-]
+[tool.poetry.dependencies]
+python = ">=3.9,<4"
+# Required for aws-cdk.aws-lambda-python-alpha
+aws-cdk-lib = ">=2.66.0,<3.0.0"
+#aws-cdk-lib = "2.48.0"
+boto3 = "^1.26.78"
+"aws-cdk.aws-lambda-python-alpha" = "^2.66.0a0"
+constructs = ">=10.0.0,<11.0.0"
+black = "^23.1.0"
+
+[tool.poetry.group.tests.dependencies]
+# Installed by default, can be installed without these dependencies using `poetry install --without tests`
+pytest = "==6.2.5"
+moto = "^4.1.3"
+hypothesis = "^6.68.2"
```

Comment:
I didn't see hypothesis in any requirements.txt files - but I needed to install if for the tests to run. I put it under the tests dependency, but this was the only dependency I didn't pull from the previous pyproject.toml or requirements.txt files.

Reply:
Hm... I don't think this is needed. This should be brought in via other dependencies since we don't have it anywhere in ours from what I can see either.

Reply:
I don't see it anywhere in the poetry dependencies tree, if I don't install it manually. When checking the other development branch and using the pip install method, I also don't see it in the pip dependencies.

Reply:
and it is needed for you to locally run the tests locally? Can you get a traceback of which library the import is coming from when it fails?

Reply:
Yeah, I'll check it out

Reply:
You know what, my tests don't fail locally anymore. I removed the dependency. I must have been missing something in my first version